ot-node.js

```diff
@@ -991,6 +991,15 @@ class OTNode {
                 response: res,
             });
         });
+
+        server.get('/api/imports_info', (req, res) => {
+            log.api('GET: import_list.');
```

Comment:
can we have friendly sentence "GET: List imports request received." in case you make one more commit plz?